### ISSUE TITLE
Acquiring sentence probability when serving on OPT.

### ISSUE THREAD

**Issue by TZWwww** [alpa-projects/alpa#889]
I'm now serving OPT to complete some tasks.
I wonder if I can acquire the sentence LM probability for any input sentences.

I don't know if it's possible under the current version.
please inform me some idea of this.
Really appreciate that.

**Comment by merrymercy** [comment]
You should be able to get the logits from here https://github.com/alpa-projects/alpa/blob/06cb62f69b84e6607aa79d3692d5352c65ec8620/examples/llm_serving/model/wrapper.py#L480-L482
